workflows/Create_BMC_Incident_for_Top_Remediations_from_InsightVM/workflow.spec.yaml

```
@@ -0,0 +1,17 @@
+extension: workflow
+product: ["insightconnect"]
+name: Create_BMC_Incident_for_Top_Remediations_from_InsightVM
+title: "Create BMC Incident for Top Remediations from InsightVM"
+description: "This workflow will get the top remediations from InsightVM, and create a BMC Remedy ITSM Incident for each remediation. This Incident will have details on the effected endpoints as well as the Fix and risk score."
```

This workflow will get the top remediations from InsightVM, and create a BMC Remedy ITSM Incident for each remediation. The Incident will have details on the affected endpoints as well as the Fix and risk score.
```suggestion
description: "This workflow will get the top remediations from InsightVM, and create a BMC Remedy ITSM Incident for each remediation. The Incident will have details on the affected endpoints as well as the Fix and risk score."
```

Make sure this description is set as the value of `description` in the `.icon` file.